SConstruct

```diff
@@ -40,11 +46,13 @@ Default('./build', './release')
 
 # Additional mode options
 if mode == 'cache':
-    CacheDir(cache_dir)
-
-# Print the state of the repo at end of SCons run
-finish_command = Command( 'state_of_repo.log', [], gslab_scons.misc.state_of_repo, MAXIT=10) # From http://stackoverflow.com/questions/8901296/how-do-i-run-some-code-after-every-build-in-scons
-Depends(finish_command, BUILD_TARGETS)
-env.AlwaysBuild(finish_command)
-if 'state_of_repo.log' not in BUILD_TARGETS: 
-    BUILD_TARGETS.append('state_of_repo.log')
+    (cache_dir)
+
+debrief_env = {'MAXIT' : 10,
```

Comment:
But is it really being used a Scons build step? I propose we change the arguments of https://github.com/gslab-econ/gslab_python/blob/dev_four/gslab_scons/misc.py#L14 also.